main.nf

```diff
@@ -244,16 +244,16 @@ process picard {
     module 'picard/2.0.1'
     module 'BEDTools'
     module 'samtools'
-    
+
     cpus 2
     memory { 16.GB * task.attempt }
     time { 24.h * task.attempt }
     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
     maxRetries 3
     maxErrors '-1'
-    
+
     publishDir "${params.outdir}/picard", mode: 'copy'
-    
+
     input:
     file bam_picard
     set val(prefix) from name_for_picard
```

Comment:
Hi,

bed_total is the bed file before duplication removal which is generated
from bwa.

I have fixed a new version of main.nf. Hopefully it will work!

On Thu, Jun 16, 2016 at 4:13 PM, Phil Ewels <notifications@github.com>
wrote:

> In main.nf
> <https://github.com/SciLifeLab/NGI-ChIPseq/pull/4#discussion_r67351472>:
>
> >      publishDir "${params.outdir}/picard", mode: 'copy'
> > -
> > +
> >      input:
> >      file bam_picard
> >      set val(prefix) from name_for_picard
>
> Presumably need file '*.dedup.sorted.bed' into bed_total below?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SciLifeLab/NGI-ChIPseq/pull/4/files/e836f3f74c85ac3dd83cf378873ed5deab42205d#r67351472>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARdKNKxGI-_gunCzuquGWOhY4tmyHd_Yks5qMVoCgaJpZM4I3XJD>
> .
>